clients/spark/core/src/main/scala/io/treeverse/gc/UncommittedGarbageCollector.scala

```diff
@@ -270,6 +276,7 @@ object UncommittedGarbageCollector {
       "success" -> success,
       "first_slice" -> firstSlice,
       "start_time" -> DateTimeFormatter.ISO_INSTANT.format(startTime),
+      "start_list_time" -> DateTimeFormatter.ISO_INSTANT.format(startTime),
```

Comment:
Probably a dumb question, but why do we need both? What does `start_list_time` give us that `start_time` don't?